tests/installation/partitioning/msdos_partition_table.pm

```diff
@@ -22,11 +22,13 @@ sub run {
     my $partitioner = $testapi::distri->get_expert_partitioner;
     $partitioner->run_expert_partitioner;
     foreach my $disk (@{$test_data->{disks}}) {
-        $partitioner->create_new_partition_table({disk => $disk->{name},
-                table_type => $disk->{table_type}, accept_deleting_current_devices_warning => 1});
+        $partitioner->create_new_partition_table({name => $disk->{name},
+                table_type => $disk->{table_type}, accept_deleting_current_devices_warning => 0});
```

Comment:
you don't need to pass it if it is 0, will be undef inside the method. It is more clear only pass it when need it.

Reply:
it was failing with 1 if I remember well